src/test/java/com/chat/yourway/integration/service/impl/ContactServiceImplTest.java

```diff
@@ -1,64 +1,82 @@
 package com.chat.yourway.integration.service.impl;
 
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-
 import com.chat.yourway.dto.request.ChangePasswordDto;
 import com.chat.yourway.exception.NoEqualsPasswordException;
+import com.chat.yourway.integration.extension.PostgresExtension;
+import com.chat.yourway.integration.extension.RedisExtension;
 import com.chat.yourway.model.Contact;
 import com.chat.yourway.repository.ContactRepository;
 import com.chat.yourway.service.ContactService;
-import org.junit.jupiter.api.BeforeEach;
+import com.github.springtestdbunit.annotation.DatabaseSetup;
+import com.github.springtestdbunit.DbUnitTestExecutionListener;
+import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.security.crypto.password.PasswordEncoder;
-import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.support.DependencyInjectionTestExecutionListener;
+import org.springframework.test.context.support.DirtiesContextTestExecutionListener;
+import org.springframework.test.context.transaction.TransactionalTestExecutionListener;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
 
+@ExtendWith(PostgresExtension.class)
+@ExtendWith(RedisExtension.class)
 @SpringBootTest
-@ActiveProfiles("test")
-@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@TestExecutionListeners(value = {
```

Comment:
Also you can add MockitoTestExecutionListener and ResetMocksExecutionListener for future (now they will do nothing)